pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -918,6 +918,28 @@ type Worker struct {
 	// SystemComponents contains configuration for system components related to this worker pool
 	// +optional
 	SystemComponents *WorkerSystemComponents `json:"systemComponents,omitempty" protobuf:"bytes,18,opt,name=systemComponents"`
+	// Settings contains configurations for different worker-pools. Eg. MachineDrainTimeout, MachineHealthTimeout.
+	// +optional
+	Settings *Settings `protobuf:"bytes,19,opt,name=settings"`
+}
+
+// Settings contains configurations for different worker-pools.
```

Comment:
Please refine

Reply:
done - https://github.com/gardener/gardener/commit/9aacdeb0348a4d06173bb54d71af652807da0846#diff-9a667891729b65e0ebbf3feef920fcd9R761 . Please suggest, if anything specific need to be added.